frontend/src/components/search/SearchOpportunityStatus.tsx

```diff
@@ -1,30 +1,51 @@
 import React from "react";
+import { Checkbox } from "@trussworks/react-uswds";
 
 // TODO: import uswds checkbox
 // There's a variant for checkbox to display it in a box
```

Comment:
👍

Reply:
Nixed this comment now that we're using the component/variant